<?xml version="1.0"?>
<launch>
	<node pkg="cv_camera" type="cv_camera_node" name="cv_camera" output="screen">
		<param name="image_width" value="640" />
		<param name="image_height" value="360" />
	</node>

    <param name="package_name" value="record" />
    
    <param name="take_name" value="take1-16" />
    
    <!-- robotis op3 manager -->
    <include file="$(find op3_manager)/launch/op3_manager.launch"/>

	<node pkg="record" type="read_serial.py" name="read_serial" output="screen">
	</node>


	<node pkg="record" type="record.py" name="record" output="screen">
	</node>

	<node pkg="op3_gripper" type="gripper.py" name="gripper" output="screen">
    </node>
</launch>

